MapboxNavigationTests/Sources/Tests/DummyURLSessionDataTask.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+class DummyURLSessionDataTask: URLSessionDataTask {
```

Comment:
Is this used?
If so, please stick to the established indentation style and remove the empty comments.

Reply:
yes this is used